src/core/server/server.cc

```diff
@@ -943,7 +1163,7 @@ void Server::Start() {
     }
   }
   for (auto& listener : listeners_) {
-    listener.listener->Start(this, &pollsets_);
```

Comment:
It seems odd that we don't need to plumb the pollsets down into the listener anymore.  I see that none of the existing impls were actually using this parameter, though.  How is this actually working?  I assume we still need pollset linkage until we eliminate the `poll` polling engine.